examples/docs_beta_snippets/docs_beta_snippets/guides/monitor-alert/custom-logging/asset-job-example.py

```diff
@@ -0,0 +1,38 @@
+import typing
+
+import requests
+
+import dagster as dg
+from dagster import json_console_logger
+
+
+@dg.asset()
+def hackernews_topstory_ids(context: dg.AssetExecutionContext) -> typing.List[int]:
+    """Get up to 500 top stories from the HackerNews topstories endpoint.
+
+    API Docs: https://github.com/HackerNews/API#new-top-and-best-stories
+    """
+    newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+    top_500_newstories = requests.get(newstories_url).json()
+
+    # Log the number of stories fetched
+    context.log.info(f"Compute Logger - Got {len(top_500_newstories)} top stories.")
+    return top_500_newstories
+
+
+def with_default_logger(config: dict) -> dict:
+    if not config.get("loggers"):
+        config["loggers"] = {"console": {"config": {"log_level": "WARNING"}}}
+    return config
+
+
+hackernews_topstory_ids_job = dg.define_asset_job(
+    name="topstory_ids_job",
+    config=with_default_logger({}),
+)
```

Comment:
I found the indirection of `with_default_logger` to be unintuitive for a simple example. Could we consider providing the config directly?
```suggestion
hackernews_topstory_ids_job = dg.define_asset_job(
    name="topstory_ids_job",
    config={
        "loggers": {
            "console": {
                "config": {
                    "log_level": "WARNING"
                }
            }
        }
    },
)
```

Reply:
i could. this is however to circumvent a major limitation of configurability of this feature - you have to set this on all jobs. 

i often found that for this as well as k8s resource config, not all users have the intuition to do something like this and when provided with this simple solution, it placates their frustration

Reply:
can we just have this be a constant? e.g.:

```python
LOGGER_CONFIG = {"loggers": {"console": {"config": {"log_level": "WARNING"}}}}

hackernews_topstory_ids_job = dg.define_asset_job(
    name="topstory_ids_job", config=LOGGER_CONFIG
)
```

if people want to combine this to some other config, they can always just merge the dictionaries together, e.g. `config = {**special_config, **LOGGER_CONFIG}`